app/jobs/reports/monthly_key_metrics_report.rb

```diff
@@ -32,19 +53,29 @@ def emails
       emails
     end
 
-    def monthly_key_metrics_report_array
-      csv_array = []
+    def account_reuse_queries
+      @account_reuse_queries ||= Reporting::AccountReuseAndTotalIdentitiesReport.new(report_date)
+    end
```

Comment:
its not really queryies anymore, looks like we landed on report, so I'd rename to match
```suggestion
    def account_reuse_report
      @account_reuse_report ||= Reporting::AccountReuseAndTotalIdentitiesReport.new(report_date)
    end
```